examples/employee-app/src/main/public/index.html

```diff
@@ -0,0 +1,189 @@
+<!doctype html>

```

Comment:
The indentation needs to be fixed in this file.

Reply:
Reindented the file with Netbeans. Should be fixed.